definitions/checks/maintenance_mode/check_consistency.rb

```diff
@@ -25,8 +25,8 @@ def run
     def verify_with_features
       procedure_arr = []
       feature_status_msgs = []
-      is_mode_on = feature(:iptables).maintenance_mode_chain_exist?
-      [:iptables, :sync_plans, :cron].each do |feature_name|
+      is_mode_on = feature(:instance).firewall.maintenance_mode_status?
+      [feature(:instance).firewall.label, :sync_plans, :cron].each do |feature_name|
```

Comment:
How expensive is it to determine `feature(:instance).firewall`? Should this be stored in a separate variable and reused?